doc/src/tools/kani.md

```diff
@@ -84,22 +84,22 @@ pub mod verify {
 }
 ```
 
-### Step 2 - Run the Kani verify-std subcommand
+### Step 2 - Run the Kani script on the std library
 
-To aid the Rust Standard Library verification effort, Kani provides a sub-command out of the box to help you get started.
-Run the following command in your local terminal (Replace "/path/to/library" and "/path/to/target" with your local paths) from the verify repository root:
+To aid the Rust Standard Library verification effort, Kani provides a script out of the box to help you get started.
+Run the following command in your local terminal from the verify repository root:
 
 ```
-kani verify-std -Z unstable-options "/path/to/library" --target-dir "/path/to/target" -Z function-contracts -Z mem-predicates
+./scripts/run-kani.sh --path .
 ```
 
-The command `kani verify-std` is a sub-command of the `kani`. This specific sub-command is used to verify the Rust Standard Library with the following arguments.
+To pass kani arguments such as `--harness`, you can run the script with `--kani-args` and continue passing in all the necessary arguments:
 
-- `"path/to/library"`: This argument specifies the path to the modified Rust Standard Library that was prepared earlier in the script. For example, `./library` or `/home/ubuntu/verify-rust-std/library`
-- `--target-dir "path/to/target"`: This optional argument sets the target directory where Kani will store its output and intermediate files. For example, `/tmp` or `/tmp/verify-std`
+```
+./scripts/run-kani.sh --path . --kani-args --harness alloc::layout::verify::check_array_i32 --output-format=terse
+```
 
-Apart from these, you can use your regular `kani-args` such as `-Z function-contracts`, `-Z stubbing` and `-Z mem-predicates` depending on your verification needs. If you run into a Kani error that says `Use of unstable feature`, add the corresponding feature with `-Z` to the command line.
-For more details on Kani's features, refer to [the features section in the Kani Book](https://model-checking.github.io/kani/reference/attributes.html)
+The script `run-kani` installs the right version of Kani for you, builds it and then finally runs the verify-std sub-command of the `kani` with some default flags.
```

Comment:
I wonder if we should put a note here about the `brew link` error people encounter for different versions of CBMC? I know it's not a fault of the script, but a sentence or two to the effect of "if the script fails with this error message, link the new CBMC version and re-run" may be useful. I don't feel strongly, though.

Reply:
Yes, we should. I also have a small fix/error handling for that error that we can add alongside. But for this PR, I'll just add a note about that error.